packages/components/checkboxes/checkboxes.js

```diff
@@ -1,73 +1,114 @@
 const { toggleConditionalInput } = require('../../common')
 
 /**
+ * Checkboxes component
+ *
  * Conditionally show content when a checkbox button is checked
```

Comment:
should this comment be moved to the syncAllConditionalReveals method on line 55?